ConservationInternational/resilienceatlas
# feat: Adding new attributes to Layer model - analysis_type

This PR tweaks some Layer attributes. Mainly it:
 - removes `timeline_overlap` which we have agreed that we don't know what it is used for
 - adds `analysis_type` where user can pick used graph/text
 - adds `analysis_text_template` where user defines used text as template (used when `analysis_type` == text)
 - adds validation that ensures that only valid analysis type for cog layer provider is histogram
 - adds validation that ensures that only valid analysis type for cartodb (vector data) is text

Unfortunately, all translated text needs to be inside same block so I had to move `analysis_text_template` outside of analysis block and put it inside Translated fields section.

<img width="1011" alt="Screenshot 2023-05-05 at 14 45 38" src="https://user-images.githubusercontent.com/20660167/236461056-c6507514-95f6-4bb6-b6ce-62e9cbc475b0.png">


## Testing instructions

Open backoffice and verify that you can set up new analysis type for selected layer. Make sure that validations works as mentioned at description of this PR.

## Tracking

https://vizzuality.atlassian.net/browse/RA2-226
https://vizzuality.atlassian.net/browse/RA2-230
https://vizzuality.atlassian.net/browse/RA2-216
